dir/path_linux.go

```diff
@@ -0,0 +1,15 @@
+//go:build linux
+// +build linux
```

Comment:
We don't need to specify `linux`. This is considered as general implantation.

Reply:
We probably need to rename the file name to `path_unix.go`.

Reply:
Try reading this doc to learn more: https://www.digitalocean.com/community/tutorials/building-go-applications-for-different-operating-systems-and-architectures#using-goos-and-goarch-filename-suffixes

Reply:
Updated.